Sources/SpeziAccount/Views/AccountOverview/NameOverview.swift

```diff
@@ -30,6 +30,7 @@ struct NameOverview: View {
                 Section {
                     NavigationLink {
                         wrapper.accountKey.singleEditView(model: model, details: accountDetails)
+                            .anyViewModifier(service.viewStyle.securityRelatedViewModifier)
```

Comment:
Should we remove the `.anyViewModifier(service.viewStyle.securityRelatedViewModifier)` at the bottom of the view here?